build.gradle

```diff
@@ -225,13 +225,24 @@ def initJsonParameters()
   project.ext.set("jsonParameters", jsonParameters)
 }
 
-def getVariableValue(String name, defaultValue)
-{
-  def value = project.hasProperty(name)
-          ? project.property(name)
-          : project.jsonParameters[name] ?: defaultValue
+def getVariableValue(String name, defaultValue) {
+  def envValue = System.getenv("OZGH_${name.toUpperCase()}")
+
+  if (envValue != null) {
+    return envValue
+  }
+
+  if (project.hasProperty(name)) {
+    return project.property(name)
+  }
+
+  def jsonValue = project.jsonParameters[name]
+
+  if (jsonValue != null) {
+    return jsonValue
+  }
 
-  return value
+  return defaultValue
```

Comment:
A colleague just made the following suggestion: In case of multiple sources for a variable, the priority is currently:

1. environment variable
2. arguments to gradlew
3. values set in the json file

We think this is less intuitive and would prefer a system where the most explicit would take priority over less explicit variables. Could you change the priority to:

1. arguments to gradlew (i.e. the `project.hasProperty` call)
2. values set in the json file
3. environment variables

Thanks!